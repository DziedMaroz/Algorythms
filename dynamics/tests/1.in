2429818498854
692
933352
4
8
3837866458
4
46488546753
85198
478646
1
59330
387
601
503795

458595776
08
3528698
285

2827677358
83476887
73
22636600
714
252
770361939
1125363805
2611780795
823951574709
61100
490201705
598705413470
21982368836
564
40196327066
9767272979
8829
5356
313621483248
307292810844
0984

327671907
8384969727
7
47458
639622
05
51574
449
4
497487597
831

753574040
21
452796434049
02428
178827
80757651762
54220081700
0394865595
5054628
16729
31488611220

7810899019
72027065511

01416163693
169514
43432722224
301836885
18
58427497

9152855
47991959

67
26337
27
921
4340409
61
470123310553
25
948
727578637576
6654106
720178692643
86
21373725761
71985
76803949780
347059379200
730818275224
4319777
2007453004
01348171045
84

3
2452
50163
409946481204
89776555
6300
73973103
599346347
9803

313432
84
312816
880276178837
89816
99052
2027
13286230257
87
39248484645


222780926749
726301
6113325581
6425
86930
3366
7463628
207605
238549314193
47
78758
4621660081
5884415780
138070321
0041
149304790972
2026876767
283479
77927333147
25678699
96485172
12413989594
5271125
00069949644
280315782
4651014
80053
0379
5
740671
7283
788747897
69325815
72402330430
9708158
210440
265188
9172
77271
834
471580656
155413280
09951638876
38
463660153
597
518841
61
612976273043
6126
52
5862651
9688
4576
954233
622
37
959821066830
3
9869645
1430
359243245
9394

5
90588782985
8429757939
6911
071654301295

2673
84759387857
385951715
783354864
82
32279
5847480
35112482
592629620416
04
777095

38560
7652791317
1911
849865
596748004
1125538
09
5836602654
8
0
8
08
243247
3516247

3
192059812280
03160209
299
0195544782
8484
45
25688231614
3161988968
611175167031
4

3630982
773
183463562
1596
942544956297
475526642
3518730
8352495
13918194
46747542
81719704
2528732733
47
3
77945
79633371
558
054
9560
1299549314
992995
876700235972
2463
51098
51448894320
1467
4220343746
57741
7034
006701720304
507848377
6
60067656662
5608
464578958207
5115517
1021929495
159948950344
42
152982441310

470
58
6962542
9805922
439439471
0996455090
57900704521
7148
91572
838

5437643

807078201

871703
42611
3817046009
34099195619
674483592638
54
7894620
028713615
898
03546059216
80
35324532529
3

58000002
920849347477
9088866

29257
060263833
53487988
46876
27056988
1377895
45879162
65
75274
6
421852999592
7
0
776241
56

79177236
17126539
027
4043327
34247352147
761273145
071170484
1
268062
586770997598
20038
3653942

921280539642
1048054570
06
2
7
94190633
8804
15
21618824541
58082
788487887117

882635474
637232688557
0602
419683081828
6974
44372645063

436
35479
42610639
571525575
053997833
5125
84645
888
09
376782
5442737
5876931
0366493
673
66820

759396
9488363511
1433597825
14
209437
45744557
1011031
7200
602165652
41145706
68276933943
16245319
29863203
943102626

0
12
8270755634
347046
52002

75280
300547237
098199419
04558
9941

537
715194
065275235200
0147291
6651
349402946393
8
6916819
82786944836
81633560
6569439338
081638
3216
0355361655
5004
053465408
53
38724388686
7655
8739
056799558
145822
2259748

59319005
1680260
38070210
0514545115
2388766
771404
633918127
5
113128600235
303111082
20
491
371
46753047169
211882020
3
02
4600
77319
06107977
0066212
6
30
8413
0369019173
31887762666
7019338
5
29
46638427693
83475503
550882592653
0578194
4561244405
14237840
18519792271
610254906
1030320743
3128068
727031437228
196
43
372412411
105017880

8966
0217
98612
596031
578648477

11533489
66343
9865
2
551726926
05639314
12012
988680802
71344297134
97323
8583005677
47213083003
4369923
0
29070588
986097616
2117386050
424

198964014
701714114
57
9
1003535533
3042317
5547
613069
5
27325972571
41556
760583659757
91158280027
41429063


48495923
915037652
966976583
9484
5631

88198251038
82713867
469377061
45
665861801100
673726258642
49134
5
9228899
7
027608222160
854
835
770593386

25062242
43828
868900553
114
113
53069927827
85
67763258432
340700405
00105828164
126447482952
11737
22194
690
6775082842

1819872
770819242011
842
352052826
70956210975
78266035

152028599
25791
3969711086
8
7115
8684710
3263
371804867274
909347264212
17612944
416030502
140610
1
94985
79111696
790514
332173946
46779713
917
2005
6426707949
8
8836879753
1602663738
1254691
8308800
90479612
97073921
198296922914
230459473577
43782344380
6481
28832587627
553924
35053
33268
118594451
43245
832641538233
3
62236
52362
0231216

7719566
58350691319
552308402

58068288
48161
51087166871
983
8301040

852
933288
8878
495
06021391
1553
35
5322341199
19224442
5

61558
28875250
92107861309

454921528
44321
509038346191
255
611763
28200
979464
8987253880

3143609
3454
28868
43710
62481774580
48
9080
677785662
128

61041
62728118
1946364586
63155296
0014144903

512
7536977854
31889065694
75
6167
4239468960
77632
7
183206
7018176114
0127257
68681
113
0765709120
6593167
543282449
1
562377
34844

799946394
655
0523300925
351
654187061
774
266
75
0768683
340
0231879

890188400175
421426995640
281
190665903
8806971837

86645456229
36833135
05857179
76044187953
698984661621
00872144
3883732309
50212
5805644049
