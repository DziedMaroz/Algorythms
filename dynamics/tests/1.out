9
2 4 29 8 1 84 9 8 854
